site/projects/isentropic_analysis.md

```diff
@@ -10,16 +10,29 @@ _description here_
 ## Related Training Notebooks
 
 ### Data Access
-* Siphon Access & Parsing
-* Downloading Model Data with NCSS
+* Siphon (TDS intro)
+* Siphon (catalogs basic)
+* Siphon (catalogs filtering)
+* Siphon (remote_access)
+* Siphon (remote_open)
+* Siphon (subset)
 
 ### Analysis
-* xarray basics (data access, indexing, and interpolation)
-* xarray intermediate (aggregations and calculations)
+* xarray  (data access)
```

Comment:
Extra space after xarray

Reply:
"find" + "replace all" is good, when the replace with is correct and does not contain extra spaces! 😆 Fixed now.